src/app/modules/pictureapp-explore/components/edit-userProfile/components/edit-user-photos/edit-user-photos.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
[1] Why do we use test components in production code? Is this by convention related to angular coding style?

Reply:
its basic ui-test file generating with components. In this file you can check if exists expected function or html generating by self component. The spec file should be close to testing components its typical angular structure